HACKING.md

```diff
@@ -17,6 +17,26 @@ On Debian/Ubuntu:
 
     $ sudo apt install nodejs
 
+
+Creating VM images locally (not necessary for running tests) needs the
+following:
+    
+    $ sudo apt-get install virtinst
+
+For running integration tests, the following dependencies are required:
```

Comment:
This list already exists, https://github.com/cockpit-project/cockpit/blame/main/HACKING.md#L41

Adjust there is some package is missing.

Reply:
Should I include just an alternative apt-get command for debian then? Or just omit it out

Reply:
An `apt install` list would be nice indeed.

Reply:
An `apt install` list for running the integration tests right?